tox.ini

```diff
@@ -10,30 +10,24 @@ python =
 deps =
     -rrequirements.txt
     -rrequirements-dev.txt
-whitelist_externals =
-    make
-    mkdir
-    rm
-passenv = TOX_ENV_DIR
 usedevelop = true
 
 commands =
     pytest \
-        --cov-config .coveragerc --cov=pyarn --cov-report term \
-        --cov-report xml --cov-report html
+        --cov-config .coveragerc --cov=pyarn --cov-report term
 
 [testenv:black]
 description = black checks [Mandatory]
 skip_install = true
 deps =
-    # Pin the version of black and click to avoid a newer version causing tox to fail
+    # Pin the version of black to avoid a newer version causing tox to fail
     black==22.6.0
-    click==8.0.3
 commands =
     black --check --diff pyarn tests
 
 [testenv:isort]
 skip_install = true
+profile = "black"
```

Comment:
Careful, `[testenv:isort]` is Tox configuration, not isort configuration. The configuration for isort would go in `[isort]` or a separate project file (like setup.cfg)

https://pycqa.github.io/isort/docs/configuration/config_files.html